modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -1553,6 +1553,14 @@ void TFImporter::populateNet(Net dstNet)
             layer_id[name] = id;
             connectToAllBlobs(layer_id, dstNet, parsePin(layer.input(0)), id, layer.input_size());
         }
+        else if (type == "Selu")
+        {
+            layerParams.set("scale", 1.0507009873554804934193349852946f);
```

Comment:
Could you please provide an origin of these magic values? Are they results of some optimization problem that we can reproduce locally or they are just some irrational numbers that have too complicated formula? This way it'd be nice to put a reference to a some proof that they are optimal.